src/main/java/com/devshop/devshop/model/OrderItem.java

```diff
@@ -0,0 +1,79 @@
+package com.devshop.devshop.model;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+public class OrderItem {
+
+	@Id
+	@GeneratedValue(generator = "orderitemSeq")
+	@SequenceGenerator(name = "orderitemSeq", sequenceName = "orderitem_seq", allocationSize = 1)
+	private long id;
+	private int amount;
+
+	@ManyToOne(targetEntity = Order.class)
+	private Order orderId;
+
+
+	@ManyToOne(targetEntity = Product.class)
+	private Product productId;
+
+	public OrderItem(int amount) {
+		this.amount = amount;
+	}
+
+
+	// na wszelki wypadek konstruktor dla wsyzstkich zmiennych
+	public OrderItem(int amount, Order orderId, Product productId) {
```

Comment:
To co ostatnio. Przekazujemy order i product a nie ich idki, czyli: 
`public OrderItem(int amount, Order order, Product product)`